pkg/common/common.go

```diff
@@ -13,6 +13,8 @@ const (
 	CDI_VERSION = "v1.0.0"
 
 	IMPORTER_DEFAULT_IMAGE = "docker.io/kubevirt/cdi-importer:" + CDI_VERSION
+	//CLONER_DEFAULT_IMAGE = "docker.io/kubevirt/cdi-cloner" + CDI_VERSION
```

Comment:
Delete from final PR

Reply:
Do I need specific permissions to push to docker.io/kuvebirt? Should I do push it or someone who reviews it and approve it?

Reply:
Images are only released by the CI, no manual release should be done.